cumulusci/tasks/bulkdata/mapping_parser.py

```diff
@@ -123,13 +135,14 @@ def _get_permission_type(self, operation: DataOperationType) -> str:
         return "createable"
 
     def _check_object_permission(
-        self, global_describe: Dict, sobject: str, operation: DataOperationType
+        self, global_describe: Mapping, sobject: str, operation: DataOperationType
     ):
+        assert sobject in global_describe
```

Comment:
`global_describe` will not contain `sobject` if the user doesn't have Read permission at the object level.

Reply:
That case should be caught on line 235 which is new.

https://github.com/SFDO-Tooling/CumulusCI/pull/1919/files/98a4287b36aa7070c85cfb4896c2baf279f999de#diff-0a8fc3e0cff0ad9e099cacf4f6fb0f41R235

Reply:
Ah, got it. Thanks for clarifying.